atlasdb-perf/src/main/java/com/palantir/atlasdb/performance/benchmarks/table/ConsecutiveNarrowTable.java

```diff
@@ -150,4 +151,9 @@ private static void storeDataInTable(ConsecutiveNarrowTable table, int numOverwr
         return value;
     }
 
+    public static Cell getRandomCell() {
+        Random generator = new Random();
```

Comment:
you don't want to create a new random every time this is called, you should be able to make Random an instance field